singularity/push.sh

```diff
@@ -16,9 +16,6 @@ export RABBITMQ_LOG_BASE=/data/dpdash/rabbitmq
 # vars for celery worker
 export dppy_config=/data/dpdash/configs/dppy.conf
 
-# vars for DPdash node app
```

Comment:
I am sure you detected it right but can you explain why the two variables were here before and why don't we need them anymore? Zoom call okay if that is easier ...

Reply:
They were used only by the old "configuration import" feature, and my replacement code doesn't need them.

`DPDASH_UPLOADS` was an upload directory for Node.JS to write to the filesystem. The old feature saved the CSV to that directory before processing it. My code just takes in JSON and processes it in-memory.

`DPDASH_UPLOADS_CONFIG_SCHEMA` has been deprecated by my function `getConfigSchema`. My reasoning for moving this into code is that the needed fields for a configuration are already hard-coded, so a schema to match that can also live in code alongside that. If the schema changes, then the whole architecture of configurations would also need to change.